basic-principles.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SQL 开发规范及基本原则
+category: principle
```

Comment:
Remove `category` and add a `summary` line.

Reply:
added